content/blog/vlan-versus-vxlan.md

```diff
@@ -0,0 +1,93 @@
+---
+title: VLAN Versus VXLAN
+date: 2024-11-11T14:57:37.063Z
+author: Indira Manthri
+authorimage: /img/indira_manthri.jpg
+disable: false
+tags:
+  - aruba
+  - aruba-central
+  - networking
+---
+This technical blog post gives insight on the need for VXLAN over VLAN. It is interesting to compare these two features as they look similar from top level but there is lot of difference in the networking layer they are used.  
+
+Lets dive into the definition of VLAN and VXLAN, details of the VLAN and VXLAN headers in the frame and the sample configuration provided in this blog post for inquisitive community members.
+
+### What is VLAN?
+
+VLAN – Virtual Local Area Network – Layer 2 which segments a physical network into multiple isolated networks. This is achieved by VLAN-ID which can range from <1-4096> which is 12 bit. Different ports on a networking device can be associated to different VLAN-IDs and the networking device would ensure that the incoming/outgoing traffic is forwarded based on the VLAN-ID which is present in the ethernet frames.
+
+So, if the physical network is layer-1, the physical network segmented using VLANs is called layer-2 data link layer.
+
+### What is VXLAN?
+
+VXLAN – Virtual Extensible Local Area Networks – Layer 2 over Layer 3 network
+VXLAN segment is identified by VNI (Virtual Network Identifier) which is 24 bits. The layer-2 ethernet frames are encapsulated by VNI outer headers. Due to this encapsulation, VXLAN is also called tunneling. The VXLAN tunnel end point is called VTEP. VTEPs encapsulate and decapsulates the ethernet frames into VXLAN packets. The intermediate networking devices receiving these VXLAN packets are routed as any other Layer3 Packet. 
+
+### Why do we need VXLAN?
+
+With VLANs being used for segmentation across and within the data centers, spanning tree protocol (STP) will be used for forming loop-free topology. STP disables few links which also results in large number of disabled links.
+This issue would be solved by using a Layer-3 virtual tunnel over the layer-2 network.
+
+A 12-bit VLAN ID is used in the Ethernet data frames to divide the larger Layer 2 network into multiple broadcast domains.  VLANs could be used only for data centers that require fewer than 4094 VLANs. So, scalability issue is resolved using VXLAN.
+
+There could be 24 or 48 servers connected to the Top-of-the-Rack (ToR) switch. Instead of just one MAC address per server link, the switch must learn all the MAC addresses of the virtual machines (VMs) communicating across the servers. This inadequate MAC address table issue would be addressed with the layer3 tunnel over the layer2 frames.
```

Comment:
In order to help this flow a little better from the above paragraph, consider rephrasing to:

"Consider a situation where there are 24 or 48 servers connected to a Top-of-the-Rack (ToR) switch. Instead of just one MAC address per server link, the switch must learn all the MAC addresses of the virtual machines (VMs) communicating across the servers. This inadequate MAC address table issue would be addressed with the Layer 3 tunnel over the Layer 2 frames."